autoload/tsuquyomi/es6import.vim

```diff
@@ -124,6 +124,11 @@ function! tsuquyomi#es6import#createImportBlock(text)
       call add(l:result_list, l:importDict)
     endif
   endfor
+
+  if g:tsuquyomi_case_sensitive_imports == 1
+    call filter(l:result_list, {idx, val -> val.identifier ==# l:identifier})
```

Comment:
I think it's a nice feature 😄 

But I don't want to use Vim's lambda syntax because some users run tsuquyomi on Vim 7.x .
So would you use not lambda but `v:val` syntax? 

```vim
call filter(l:result_list, 'v:val.identifier==#l:identifier')
```

Reply:
Good catch, will do! 🤓